authors/apps/articles/views.py

```diff
@@ -139,3 +138,93 @@ def put(self, request, pk):
                 {
                     'error': 'You cannot edit an article you do not own. '
                 }, status.HTTP_403_FORBIDDEN)
+
+
+class ArticleLikes(generics.ListCreateAPIView):
+    serializer_class = LikesSerializer
+
+    def get_object(self, pk):
+        try:
+            return Article.objects.get(pk=pk)
```

Comment:
For consistency of our application, do we use slug or pk to search for an article? I have personally used the slug.

Reply:
We have not resolved on what to use since `searching` is all different from retrieving a particular article.